buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy

```diff
@@ -29,10 +29,9 @@ class DownloadPage {
         new GuideGroup(title: "Install",
                 image: "${getImageAssetPreffix(url)}download.svg",
                 items: [
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installSdkman", title: 'Install with SDKman'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installHomebrew", title: 'Install with Homebrew'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installMacPorts", title: 'Install with MacPorts'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installWindows", title: 'Install through Binary on Windows'),
+                        new GuideGroupItem(href: "https://micronaut-projects.github.io/micronaut-starter/latest/guide/#installation", title: 'Install with SDKman'),
```

Comment:
there are no anchors, should I create anchors for these aka, make them <h>'s?

Reply:
I think we should split installation in starter into multiple sections.

Reply:
I have created the PR and assign it to you: https://github.com/micronaut-projects/micronaut-starter/pull/471